.prettierrc

```diff
@@ -0,0 +1,10 @@
+{
+  "useTabs": true,
+  "printWidth": 80,
+  "tabWidth": 2,
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "jsxBracketSameLine": false,
+  "parser": "flow",
+  "semi": true
+}
```

Comment:
There is no new line in the end of the file. Do you have [editorconfig](https://editorconfig.org/) plugin enabled in your IDE/editor? It should have inserted new line automatically.

Reply:
Fix - 0401711e811867528918a532b1e15fec48ba7cde